src/main/java/mekanism/common/tile/TileEntityBoundingBlock.java

```diff
@@ -27,7 +30,7 @@
 /**
  * Multi-block used by wind turbines, solar panels, and other machines
  */
-public class TileEntityBoundingBlock extends TileEntityUpdateable implements IUpgradeTile, ISecurityTile {
+public class TileEntityBoundingBlock extends TileEntityUpdateable implements IUpgradeTile, ISecurityTile, Nameable {
```

Comment:
Is there really much use in making the bounding blocks be Nameable, or is this just in case other mods might want to query information via them? I suspect at least for Jade and TOP this may not be needed due to redirects I already do, but if you think it is worth leaving them in we can.

Reply:
From my testing, Jade needed that in order to pickup the custom name, if it exists. I'd leave it there in case any other mod needs the block's name for whatever reason.